app/templates/client/components/auth(auth)/auth.service.js

```diff
@@ -20,7 +19,7 @@ function AuthService($location, $http, $cookies, $q, appConfig, Util, User) {
      * @param  {Function} callback - optional, function(error, user)
      * @return {Promise}
      */
-    login: function(user, callback) {
+    login: function(user, callback = angular.noop) {
```

Comment:
Yeah, this is on purpose. If a use puts in `'oogabooga'` for `callback`, we want an error, not to silently eat it.